oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnector.java

```diff
@@ -110,12 +110,15 @@ protected String getConnectionString(@Nullable String database) {
     if (database == null) {
       return config.getConnectionString();
     }
-    if (OracleConstants.SERVICE_CONNECTION_TYPE.equals(config)) {
+    if (OracleConstants.TNS_CONNECTION_TYPE.equals(config.getConnectionType())) {
       return String.format(OracleConstants.ORACLE_CONNECTION_STRING_SERVICE_NAME_FORMAT, config.getHost(),
                            config.getPort(), database);
```

Comment:
should this return
`return String.format(OracleConstants.ORACLE_CONNECTION_STRING_TNS_FORMAT, database);`

Reply:
thanks for catching this.